tensorflow/lite/tools/optimize/operator_property.cc

```diff
@@ -913,6 +913,7 @@ OperatorProperty GetOperatorProperty(const ModelT* model, int subgraph_index,
       property.inputs = {{0, {}}, {1, {}}};
       property.outputs = {{0, {}}};
       property.version = 2;
+      property.restrict_same_input_output_scale = true;
```

Comment:
The properties for ADD and SUB ops are different. Is it intended? What is the reason for that?

Reply:
Thanks for spotting! corrected

Reply:
Thank you